app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+  def update_like_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- Kindly use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_like_counter method every time a new like has been saved to database, then make the update_like_counter method private :+1: